_sass/blocks/how-it-works/_subpage_intro.scss

```diff
@@ -94,28 +94,23 @@
 }
 
 .revenues_subpage-office_text {
-
   @include span-columns(10);
 
-  @include respond-to(medium-up) {
-    @include span-columns(9);
-  }
-
   padding-left: $base-padding;
   padding-right: $base-padding-lite;
 
   p {
-    font-size: 12px;
+    font-size: 16px;
     line-height: 1.4;
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 107 to appear before line 102. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets